daprdocs/content/en/operations/monitoring/newrelic.md

```diff
@@ -0,0 +1,347 @@
+addapp-dapr               ClusterIP     None          <none>        80/TCP,50001/TCP,50002/TCP,9090/TCP 43s
+calculator-front-end      LoadBalancer  10.0.54.72    20.62.219.85  80:30953/TCP                        43s
+calculator-front-end-dapr ClusterIP     None          <none>        80/TCP,50001/TCP,50002/TCP,9090/TCP 43s
+divideapp-dapr            ClusterIP     None          <none>        80/TCP,50001/TCP,50002/TCP,9090/TCP 43s
+kubernetes                ClusterIP     10.0.0.1      <none>        443/TCP                             37m
+multiplyapp-dapr          ClusterIP     None          <none>        80/TCP,50001/TCP,50002/TCP,9090/TCP 37m
+redis-headless            ClusterIP     None          <none>        6379/TCP                            37m
+redis-master              ClusterIP     10.0.238.122  <none>        6379/TCP                            37m
+redis-slave               ClusterIP     10.0.115.184  <none>        6379/TCP                            37m
+subtractapp-dapr          ClusterIP     None          <none>        80/TCP,50001/TCP,50002/TCP,9090/TCP 43s
+```
+
+Each service ending in “-dapr” represents your services respective sidecars, while the calculator-front-end service represents the external load balancer for the React calculator front-end.
+
+6. Take the external IP address for calculator-front-end and drop it in your browser and voilà! You have a working distributed calculator!
+
+![Dapr Distributed Calculator UI](/images/nr-distributed-calculator-ui.png)
+
+## Observing all data collected from Kubernetes, Dapr and the Distributed Calculator
+
+New Relic Kubernetes Cluster Explorer is a good starting point to observe all your data and dig deeper into any performance issues or incidents happening inside of the application or microservices.
+
+![New Relic Kubernetes Cluster Explorer App](/images/nr-k8s-cluster-explorer-app.png)
+
+Automated correlation is part of the visualization capabilities of New Relic.
+
+### Pod-level details
+
+![New Relic K8s Pod Level Details](/images/nr-k8s-pod-level-details.png)
+
+### Logs in Context
+
+![New Relic K8s Logs In Context](/images/nr-k8s-logs-in-context.png)
+
+### Distributed Tracing
+
+![New Relic Distributed Tracing 1](/images/nr-distributed-tracing-1.png)
+
+![New Relic Distributed Tracing 2](/images/nr-distributed-tracing-2.png)
+
+## New Relic APM curated experience
+
+### Front-end application with New Relic language agent
+
+![New Relic APM Frontend](/images/nr-apm-frontend.png)
+
+### .Net Core Subtractor with OpenTelemetry Exporter
+
+![New Relic APM Subtractor 1](/images/nr-apm-subtractor-1.png)
+
+![New Relic APM Subtractor 2](/images/nr-apm-subtractor-2.png)
+
+## New Relic Dashboards
+
+### Kubernetes Overview
+
+![New Relic Dashboard Kubernetes Overview](/images/nr-dashboard-k8s-overview.png)
+
+### Dapr System Services
+
+![New Relic Dashboard Dapr System Services](/images/nr-dashboard-dapr-system-services.png)
+
+### Dapr Metrics
```

Comment:
right now this is a lot of screenshots one after the other. I suggest mentioning 1-3 examples of metrics that can be visualized in a dashboard with an explanation of the example. You can also add links to New Relic official docs for more information on dashboard creation and visualization.